src/defaultTranslations/relative.time.ts

```diff
@@ -296,313 +296,292 @@ export const translations: DefaultTranslations = {
+      day_in: '__count__天后',
+      day_in_plural: '__count__天后',
+      month_ago: '__count__个月前',
+      month_ago_plural: '__count__个月前',
+      month_in: '__count__个月后',
+      month_in_plural: '__count__个月后',
+      year_ago: '__count__年前',
+      year_ago_plural: '__count__年前',
+      year_in: '__count__年后',
+      year_in_plural: '__count__年后'
     }
   },
   'zh-HK': {
     translation: {
-      now: '剛才',
-      second_ago: '__count__ 秒鐘前',
-      second_ago_plural: '__count__ 秒鐘前',
-      second_in: '__count__ 秒內',
-      second_in_plural: '__count__ 秒內',
-      minute_ago: '__count__ 分鐘前',
-      minute_ago_plural: '__count__ 分鐘前',
-      minute_in: '__count__ 分鐘內',
-      minute_in_plural: '__count__ 分鐘內',
-      hour_ago: '__count__ 小時前',
-      hour_ago_plural: '__count__ 小時前',
-      hour_in: '__count__ 小時內',
-      hour_in_plural: '__count__ 小時內',
-      day_ago: '__count__ 天前',
-      day_ago_plural: '__count__ 天前',
-      day_in: '__count__ 天內',
-      day_in_plural: '__count__ 天內',
-      month_ago: '__count__ 月前',
-      month_ago_plural: '__count__ 月前',
-      month_in: '__count__ 月內',
-      month_in_plural: '__count__ 月內',
-      year_ago: '__count__ 年前',
-      year_ago_plural: '__count__ 年前',
-      year_in: '__count__ 年內',
-      year_in_plural: '__count__ 年內'
-    }
-  },
-  'zh-TW': {
-    translation: {
-      now: '剛才',
-      second_ago: '__count__ 秒鐘前',
-      second_ago_plural: '__count__ 秒鐘前',
-      second_in: '__count__ 秒內',
-      second_in_plural: '__count__ 秒內',
+      now: '現在',
+      second_ago: '__count__ 秒前',
+      second_ago_plural: '__count__ 秒前',
+      second_in: '__count__ 秒後',
+      second_in_plural: '__count__ 秒後',
       minute_ago: '__count__ 分鐘前',
       minute_ago_plural: '__count__ 分鐘前',
-      minute_in: '__count__ 分鐘內',
-      minute_in_plural: '__count__ 分鐘內',
+      minute_in: '__count__ 分鐘後',
+      minute_in_plural: '__count__ 分鐘後',
       hour_ago: '__count__ 小時前',
       hour_ago_plural: '__count__ 小時前',
-      hour_in: '__count__ 小時內',
-      hour_in_plural: '__count__ 小時內',
-      day_ago: '__count__ 天前',
-      day_ago_plural: '__count__ 天前',
-      day_in: '__count__ 天內',
-      day_in_plural: '__count__ 天內',
-      month_ago: '__count__ 月前',
-      month_ago_plural: '__count__ 月前',
-      month_in: '__count__ 月內',
-      month_in_plural: '__count__ 月內',
+      hour_in: '__count__ 小時後',
+      hour_in_plural: '__count__ 小時後',
+      day_ago: '__count__ 日前',
+      day_ago_plural: '__count__ 日前',
+      day_in: '__count__ 日後',
+      day_in_plural: '__count__ 日後',
+      month_ago: '__count__ 個月前',
+      month_ago_plural: '__count__ 個月前',
+      month_in: '__count__ 個月後',
+      month_in_plural: '__count__ 個月後',
       year_ago: '__count__ 年前',
       year_ago_plural: '__count__ 年前',
-      year_in: '__count__ 年內',
-      year_in_plural: '__count__ 年內'
+      year_in: '__count__ 年後',
+      year_in_plural: '__count__ 年後'
+    },
+    'zh-TW': {
```

Comment:
seems like something got messed up here as this locale now seems to be part of another. I'll fix this along with the merge but perhaps we should check that in the other PR as well @josundt

Reply:
Oops... Human mistake, the **zh-TW** translation was copied from old translation to the generated since **zh-TW** is not included with the node full-icu. I will fix this in the other PR.

I guess this proves the importance of full automation when generating the translations 😁 (#320).